### ISSUE TITLE
Could not find migration.rb

### ISSUE THREAD

**Issue by cheshire137** [ryanto/acts_as_votable#165]
Using version 0.11.0 in a Rails 4.1 app on Ruby 2.2.0, I get this error when I try to generate the migration:

```bash
% bundle exec rails g acts_as_votable:migration
Could not find "migration.rb" in any of your source paths. Your current source paths are:
/Users/me/code/my-rails-app/lib/templates/acts_as_votable/migration
/Users/me/.rbenv/versions/2.2.0/lib/ruby/gems/2.2.0/gems/acts_as_votable-0.11.0/lib/generators/acts_as_votable/migration/templates/active_record
```

**Comment by kpklein** [comment]
I'm getting the exact same error if you manage to fix it could you please notify me thanks.

**Comment by cheshire137** [comment]
While I didn't fix the generator error @kpklein, I did copy the bulk of the migration from https://github.com/ryanto/acts_as_votable/blob/master/lib/generators/acts_as_votable/migration/templates/active_record/migration.erb and manually created a new migration with `rails g migration create_votes`. You can see my migration at https://github.com/StartupWichita/startupwichita.com/pull/127/files#diff-7dd0b1db3ac9efbed4bbe4775fce7252. The rest of the gem seems to work as advertised!

**Comment by kpklein** [comment]
thanks for your help.

**Comment by Zitzabis** [comment]
Another method is to track down the file in
`C:\RailsInstaller\Ruby2.2.0\lib\ruby\gems\2.2.0\gems\acts_as_votable-0.11.0\lib\generators\acts_as_votable\migration\templates\active_record`


You'll find the file is named "migration.erb" instead "migration.rb".
Just change the file extension and then go back and run:
`rails generate acts_as_votable:migration`
`rake db:migrate`

**Comment by epergo** [comment]
Solved in #166
